modules/balana-core/src/main/java/org/wso2/balana/xacml3/Attributes.java

```diff
@@ -210,9 +210,9 @@ public void encode(StringBuilder builder) {
         builder.append("<Attributes Category=\"").append(category.toString()).append("\">");
 
         for(Attribute attribute : attributes){
-            if(attribute.isIncludeInResult()){
-                attribute.encode(builder);
-            }
+            //if(attribute.isIncludeInResult()){
```

Comment:
isn't this check required to decide which attribute we'll be returning in the Response?